src/language.h

```diff
@@ -1006,7 +1006,7 @@ static const char* const myLanguage[18][207] = {
     "Αναγνωριστικό", // 166
     "Αγορά", // 167
     "Λήψη δεδομένων", // 168
-    "MW default stepsize", // 169 ***
+    "Προεπιλεγμένο μέγεθος\nβήματος ΑΜ", // 169 ***
```

Comment:
I guess you may remove the `***` here

Reply:
Yes, but in my opinion it was too much to mark even lines that can never be translated (RDS etc).

Reply:
In your language yes.